app/admin/blogs.rb

```diff
@@ -0,0 +1,18 @@
+ActiveAdmin.register Blog do
+  permit_params :title, :body, :user_id
+  # See permitted parameters documentation:
+  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+  #
+  # Uncomment all parameters which should be permitted for assignment
+  #
```

Comment:
check `app/admin/admin_users.rb` and try to create a custom index and show page for blogs model